test/extended/testdata/openstack/udplb-default-manifests.yaml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
Looks like we're creating manifest in the code, do we still need this?

Reply:
Right, it was just for reference. I'm removing it.